[crossguid] Build error on Linux with gcc v13

It seems that `<cstdint>` was implicitly included in previous versions of gcc but now compilation of crossguid fails like so:

```
[1/3] /usr/lib/ccache/bin/c++ -DGUID_LIBUUID -I/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include -I/home/alex/code/vcpkg/installed/x64-linux/include -fPIC -g -std=c++17 -fPIC -MD -MT CMakeFiles/crossguid.dir/src/guid.cpp.o -MF CMakeFiles/crossguid.dir/src/guid.cpp.o.d -o CMakeFiles/crossguid.dir/src/guid.cpp.o -c /home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/src/guid.cpp
FAILED: CMakeFiles/crossguid.dir/src/guid.cpp.o 
/usr/lib/ccache/bin/c++ -DGUID_LIBUUID -I/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include -I/home/alex/code/vcpkg/installed/x64-linux/include -fPIC -g -std=c++17 -fPIC -MD -MT CMakeFiles/crossguid.dir/src/guid.cpp.o -MF CMakeFiles/crossguid.dir/src/guid.cpp.o.d -o CMakeFiles/crossguid.dir/src/guid.cpp.o -c /home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/src/guid.cpp
In file included from /home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/src/guid.cpp:26:
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp: In member function ‘std::size_t std::hash<xg::Guid>::operator()(const xg::Guid&) const’:
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:145:31: error: ‘uint64_t’ does not name a type
  145 |                         const uint64_t* p = reinterpret_cast<const uint64_t*>(guid.bytes().data());
      |                               ^~~~~~~~
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:39:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   38 | #include <iomanip>
  +++ |+#include <cstdint>
   39 | 
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:146:50: error: ‘uint64_t’ was not declared in this scope
  146 |                         return xg::details::hash<uint64_t, uint64_t>{}(p[0], p[1]);
      |                                                  ^~~~~~~~
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:146:50: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:146:68: error: template argument 1 is invalid
  146 |                         return xg::details::hash<uint64_t, uint64_t>{}(p[0], p[1]);
      |                                                                    ^
/home/alex/code/vcpkg/buildtrees/crossguid/src/08ee1b7681-f70a1f9530.clean/include/crossguid/guid.hpp:146:72: error: ‘p’ was not declared in this scope
  146 |                         return xg::details::hash<uint64_t, uint64_t>{}(p[0], p[1]);
      |                                                                        ^
ninja: build stopped: subcommand failed.
```

--- a/ports/crossguid/missing-include-cstdint.patch
+++ b/ports/crossguid/missing-include-cstdint.patch
@@ -1,0 +1,12 @@
+diff --git a/include/crossguid/guid.hpp b/include/crossguid/guid.hpp
+index 61e0f17..70966f2 100644
+--- a/include/crossguid/guid.hpp
++++ b/include/crossguid/guid.hpp
+@@ -29,6 +29,7 @@ THE SOFTWARE.
+ #include <jni.h>
+ #endif
+ 
++#include <cstdint>
+ #include <functional>
+ #include <iostream>
+ #include <array>

--- a/ports/crossguid/portfile.cmake
+++ b/ports/crossguid/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
     HEAD_REF master
     PATCHES
         warnings.patch
+        missing-include-cstdint.patch
 )
 
 vcpkg_cmake_configure(

--- a/ports/crossguid/vcpkg.json
+++ b/ports/crossguid/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "crossguid",
   "version-date": "2021-10-22",
-  "port-version": 2,
+  "port-version": 3,
   "description": "CrossGuid is a minimal, cross platform, C++ GUID library.",
   "dependencies": [
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1882,7 +1882,7 @@
     },
     "crossguid": {
       "baseline": "2021-10-22",
-      "port-version": 2
+      "port-version": 3
     },
     "crow": {
       "baseline": "1.0-5",

--- a/versions/c-/crossguid.json
+++ b/versions/c-/crossguid.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "39a8ec44dcb709796a0ecdc4c170f67ad5ad1175",
+      "version-date": "2021-10-22",
+      "port-version": 3
+    },
+    {
       "git-tree": "93714099ddf83b2f3437a080c8acc08ff74cbe37",
       "version-date": "2021-10-22",
       "port-version": 2

